nameko_amqp_retry/backoff.py

```diff
@@ -109,6 +114,10 @@ def republish(self, backoff_exc, message, target_queue):
             headers['backoff'] = expiration
             expiration_seconds = float(expiration) / 1000
 
+            # force redeclaration; the publisher will skip declaration if
+            # the entity has previously been declared by the same connection
+            maybe_declare(queue, conn)
```

Comment:
I think this will create a new connection every time, but we should have a retry policy anyway.